### ISSUE TITLE
Offbeat heavy rhythyms

### ISSUE THREAD

**Issue by hpx7** [CPJKU/madmom#446]
The beat tracker seems to struggle with music with emphasis on the offbeat.

Consider this popular song: https://www.youtube.com/watch?v=RILP53OR63k. You can hear the guitar/ukulele strumming on the offbeat. This causes `DBNDownBeatTrackingProcessor` to get confused about:
1) The tempo (it recognizes both the beats and off-beats, and thus predicts double the actual tempo)
2) The positions of the downbeats

Is this something that could be solved with more training data? Or can something be improved about the algorithm?

I noticed that the `DBNDownBeatTrackingProcessor` takes in a `min_bpm`/`max_bpm ` parameter, so I could help it out by providing it the actual bpm, but I wonder if there's anything we can do to make it work more often by default in these types of songs.

**Comment by superbock** [comment]
Yes, this kind f music is hard to get right for (down-)beat trackers since it contains strong onsets on the offbeats. Thus it often detects double the correct tempo, and in turn the downbeat positions.

There's not much you can do to "improve" the algorithm. There's nothing wrong with it. It is just that the neural network has more or less strong activations the wrong positions. This should be solvable with more training data.

The other option — as you mentioned — is to give the tracker hints about what to expect, e.g. the known tempo or hint towards known downbeat positions as suggested in #392.
